### PR TITLE
Get File's Tasks

### PR DESCRIPTION
**Pull request by marek-vysoky**
Adding implementation for GetFileTasks, adding unit test GetFileTasks_ValidResponse_ValidTasks

### CODE REVIEW

**Comment by boxcla** [comment]
Verified that @marek-vysoky has signed the CLA. Thanks for the pull request!

**Comment by coolboy** [comment]
@marek-vysoky conflict

**Comment by marek-vysoky** [comment]
I've merged master to this branch, resolved the conflict and pushed the changes

File: Box.V2/Models/BoxTask.cs [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+        public const string FieldDueAt = "due_at";
+        public const string FieldItem = "item";
+        /// <summary>
+        /// Date of task completion
+        /// </summary>
+        [JsonProperty(PropertyName = FieldDueAt)]
+        public string DueAt { get; private set; }
+
+        /// <summary>
+        ///
```
Comment (coolboy) [inline Box.V2/Models/BoxTask.cs]: Comment?

**Comment by marek-vysoky** [comment]
I've removed empty line and added comment

**Pull request merged**